docs/source/model_doc/albert.rst

```diff
@@ -44,10 +44,17 @@ AlbertTokenizer
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. autoclass:: transformers.AlbertTokenizer
-    :members: build_inputs_with_special_tokens, get_special_tokens_mask,
+    :members: __call__, build_inputs_with_special_tokens, get_special_tokens_mask,
```

Comment:
The doc still needs to be nicely done. Waiting for @sgugger feedback here. Almost no model doc files have their fast tokenizers in the doc. Also, I think all tokenizers should at least show the `__call__` method in the doc as it's the most important method. Maybe @sgugger can you give me your opinion on which methods should be shown for the slow tokenizer and which for the fast tokenizer and I'll apply it to all model docs in this PR.

Reply:
Actually, the PR is getting a bit too big already - will open a new PR to add the fast tokenizer docs and merge this one.